website/www/site/content/en/blog/2016-02-22-beam-has-a-logo.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
```

Comment:
This `layout: post` appears in every blog page. Do we need this one? I see it doesn't have any negative effects but it will confuse other maintainers. It is used in Jekyll to match each blog to the layout `post`.

Reply:
No, it's not needed, I was just too lazy to remove every weird Jekyll thing :P